.claude/commands/release-notes.md

```
@@ -1,251 +1,413 @@
+```bash
+# Check if release exists
+if gh release view "v${VERSION}" &>/dev/null; then
+  RELEASE_EXISTS=true
+else
+  RELEASE_EXISTS=false
+fi
+
+# Check if tag exists
+if git rev-parse "v${VERSION}" &>/dev/null; then
+  TAG_EXISTS=true
+else
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)